### ISSUE TITLE
Adding time dimension in the cases study

### ISSUE THREAD

**Issue by miguelherpas** [Ibercivis/odourcollect.eu#62]
A new case study in Spain started a few months ago. When we proceeded with the data analysis and downloaded the data from the BackOffice (doing the normal procedure of filtering by case study), the data was not consistent compared to the number of observations that appeared in OC. I attach here the file that we downloaded and the other one that we had done to do the proper analysis after filtering the total observations in the last months by location in excel.
[BackOffice Filtered Case Study Villarrobledo.xlsx](https://github.com/Ibercivis/odourcollect.eu/files/6677286/BackOffice.Filtered.Case.Study.Villarrobledo.xlsx)
[Villarrobledo .ods](https://github.com/Ibercivis/odourcollect.eu/files/6677288/Villarrobledo.ods)

As you can see, in the first file, there are only observations until 22/05. I think, but I am not sure and I do not know how to verify that we set the case study that day. I hope that this information gave you insights into what is happening and how to correct it.

Nevertheless, I suggest changing the way a case study works and implement the possibility of include dates to set each case study database directly. 

Thanks in advance and if you need more information, just ask us!

**Comment by frasanz** [comment]
As said, probably it's because data in the pilot starts to be collected
once  the data is defined. We can include data in the pilot "by hand", but
the good solution is to provide time-dimension to a pilot (and incorporate
odors back in time if wanted)

**Comment by miguelherpas** [comment]
I agree @frasanz, that is the best solution.
In this case, the interesting thing is that the observations stopped once the case study is defined. It is actually pretty awkward that the data that should be the most important is the one that is not included. I don't know if that info is important about the real functioning of the OC back office, but it was really surprising for me.
I hope we can find the solution quickly

**Comment by frasanz** [comment]
For the moment, If lets find the  id of the observations before the
pilot definition, I can add them to the zone 'by hand'

On the other hand, I´m going to close this issue and open a new one for
"adding time dimension" to pilots. Does that make sense?

Fran

**Comment by miguelherpas** [comment]
It makes completely sense for me. 

Thanks Fran

**Comment by vvalvesga** [comment]
about this issue then what to do?
Nor have I come to understand the matter. I see in BD that there are 2 areas of Villarobledo. The 25 and 26. The 26 is erased.
and I download the data and see 63 records, from 2021-04-15 until 2021-06-18, no 22/05

**Comment by miguelherpas** [comment]
Hello Victor, the issue was opened because there was a problem that seems to be solved now. I can assure you that when we tried to download the data for that case study last week it was not the case. In fact, when Fran provided us the data for all the case studies of D-NOSES there was an inconsistency in the values for that specific case study. 

You will probably know better than us which could have been the change that has caused that, but as I told you there was a problem regarding the data that was included in that case study. It can be related to the deleted zone, as you commented. We thought that the issue was caused by the timing in the definition of the case study, as the zone was created after the first observations were included. So we could have been wrong

Nevertheless, I think that including the time dimension in a case study can be a nice complement in OC, but we can discuss its urgency, as the main problem seems to be solved.

 Thanks for the information, btw.
